pkg/reconciler/isbsvc/controller.go

```diff
@@ -75,6 +75,9 @@ func (r *interStepBufferServiceReconciler) Reconcile(ctx context.Context, req ct
 			return reconcile.Result{}, err
 		}
 	}
+	if err := setStatefulSetStatus(r.client, isbSvcCopy); err != nil {
```

Comment:
Shall this go to `installer.go`, under `Install()`, after `isbSvc.Status.Config = *bufferConfig`?

Reply:
Not all the `InterStepBufferService` implementations have `StatefulSet`, shall we add a function to the interface `Installer` to check the children resource status, and call it in the `Install()` function?